mender/values.yaml

```diff
@@ -1,3 +1,6 @@
+nameOverride: ""
```

Comment:
Can't understand this one, it seems not used to me

Reply:
> https://github.com/mendersoftware/mender-helm/blob/master/mender/templates/_helpers.tpl#L6

It's actually already being used, but it's undocumented.. I thought, it makes sense to add it to make it more visible..

I removed it from the commit for now, and maybe it can be addressed separately.

Reply:
You're right! Thanks for bringing this up, I'll add this in a later PR